services/MerchStoreService.ts

```diff
@@ -498,6 +710,25 @@ export default class MerchStoreService {
     }, 0);
   }
 
+  public async cancelAllPendingOrders(user: UserModel): Promise<void> {
```

Comment:
Should this cancel all pending orders or only the ones that were placed a long time ago?

Reply:
This would cancel all pending orders, since there could be the case where someone orders for the last pickup event of a quarter and wasn't able to make it or wasn't able to fulfill all of it